components/dashboard/src/components/PageWithSubMenu.tsx

```diff
@@ -31,7 +31,7 @@ export function PageWithSubMenu(p: PageWithSubMenuProps) {
                             if (e.link.some((l) => l === location.pathname)) {
                                 classes += " bg-gray-300 text-gray-800 dark:bg-gray-800 dark:text-gray-50";
                             } else {
-                                classes += " hover:bg-gray-100 dark:hover:bg-gray-800";
+                                classes += " hover:bg-gray-100 dark:hover:bg-gray-800 dark:text-gray-500";
```

Comment:
While attempting to test this change, I saw no difference between:
- https://gt-change-4c330c918c.staging.gitpod-dev.com/account
- and https://main.staging.gitpod-dev.com/account

I believe the reason is that, when you look at the combined classes, `text` is _already_ `gray-500` (see `text-gray-500` in the common classes on line 28).